modules/post/windows/gather/credentials/gpp.rb

```diff
@@ -241,7 +266,19 @@ def parse_xml(xmlfile)
     tables = Rex::Parser::GPP.create_tables(results, filetype, xmlfile[:domain], xmlfile[:dc])
 
     tables.each do |table|
-      print_good table.to_s
+      # We have to manually format the results as we want to insert a new line
```

Comment:
Is there a reason that using `table << ['NAME', xmlfile[:name]] if xmlfile.member?(:name)` to add a row to the table doesn't work?

Reply:
I simply did not think about it ^^ I wanted to insert this new field on top of other values. However the method you suggested is much simpler and cleaner, and I commited your idea to the code. The "NAME" field is added so it goes to the end instead, but the code simplification is worth it. Thanks :-)

Reply:
Another (perhaps more correct option) would be to have `Rex::Parser::GPP.create_tables` take the name as an option parameter and add it to the table around https://github.com/rapid7/metasploit-framework/blob/master/lib/rex/parser/group_policy_preferences.rb#L102

Reply:
Also, thanks for making the change! The code is definitely much cleaner for it.